pkg/virt-api/rest/subresource_test.go

```diff
@@ -1390,7 +1387,7 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 					ClaimName: "vol1",
 					Phase:     v1.MemoryDumpCompleted,
 				},
-				"[{ \"op\": \"test\", \"path\": \"/status/memoryDumpRequest\", \"value\": {\"claimName\":\"vol1\",\"phase\":\"Completed\"}}, { \"op\": \"replace\", \"path\": \"/status/memoryDumpRequest\", \"value\": {\"claimName\":\"vol1\",\"phase\":\"Associating\"}}]",
+				"[{\"op\":\"test\",\"path\":\"/status/memoryDumpRequest\",\"value\":{\"claimName\":\"vol1\",\"phase\":\"Completed\"}},{\"op\":\"replace\",\"path\":\"/status/memoryDumpRequest\",\"value\":{\"claimName\":\"vol1\",\"phase\":\"Associating\"}}]",
```

Comment:
Same suggestion here to create a function that creates the assignement and the patch. In this case something like:
```golang
type memoryDumpRequest struct {
       claim string
       phase string
}


func testAndReplace(oldReq, newReq memoryDumpRequest)
```
Then the entry becomes:
```golag
testAndReplace(memoryDumpRequest{"vol1", "Completed"}, memoryDumRequest("vol1", "Associating")
```